src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -264,22 +292,70 @@ def gen_data_node(gen_data: List[str]) -> Optional[EnkfConfigNode]:
             report_steps,
         )
 
+    def add_config_node_meta(
+        self,
+        key: str = "",
+        forward_init: bool = False,
+        init_file: str = "",
+        input_file: str = "",
+        output_file: str = "",
+        var_type: Optional[EnkfVarType] = EnkfVarType.INVALID_VAR,
+    ):
+        metaInfo = ConfigNodeMeta(
+            key, forward_init, init_file, input_file, output_file, var_type
+        )
+        self._config_node_meta[key] = metaInfo
+
+    def _create_node_metainfo(
+        self,
+        keywords: Union[dict, list],
+        offset: int,
+        var_type: EnkfVarType = EnkfVarType.INVALID_VAR,
+    ):
+        options = _option_dict(keywords, offset)
+        key = keywords[0]
+        forward_init = _str_to_bool(options.get(ConfigKeys.FORWARD_INIT, "FALSE"))
+        init_file = options.get(ConfigKeys.INIT_FILES, "")
+        input_file = options.get(ConfigKeys.RESULT_FILE, "")
+        output_file = options.get("OUTPUT_FILE", "")
+
+        self.add_config_node_meta(
+            key=key,
+            forward_init=forward_init,
+            init_file=init_file,
+            input_file=input_file,
+            output_file=output_file,
+            var_type=var_type,
+        )
+
+    def _storeFieldMetaInfo(self, node_kw: Union[dict, list]):
+        name = node_kw[0]
+        var_type_string = node_kw[1]
+        out_file = node_kw[2]
+        var_type = EnkfVarType.INVALID_VAR
+        enkf_infile = ""
+        if var_type_string == ConfigKeys.GENERAL_KEY:
+            enkf_infile = node_kw[3]
```

Comment:
This block seems to not be hit by tests.

Reply:
This turns out to be unused functionality, and was removed from the docs in january 2023. This is residue leftovers which i've now removed support for.